src/components/SettingsManager/SettingsManager.UserDataSection/TickerBucketList.tsx

```
@@ -22,6 +27,35 @@ export default function TickerBucketList({
     [tickerBuckets],
   );
 
+  const { triggerUIError } = useAppErrorBoundary();
+
+  const { showNotification } = useNotification();
+
+  // TODO: Refactor to use a shared clipboard utility
+  const copySymbolsToClipboard = useCallback(
+    (symbols: string[]) => {
+      const symbolsText = symbols.join(", ");
+      if (navigator.clipboard && navigator.clipboard.writeText) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)